Sources/MongoSwift/MongoClient.swift

```diff
@@ -162,6 +167,9 @@ public class MongoClient {
     /// Default size for a client's NIOThreadPool.
     public static let defaultThreadPoolSize = 5
 
+    /// Default maximum size for connection pools created by this client.
+    public static let defaultMaxConnectionPoolSize = 100
```

Comment:
while we're here, is it useful for this to be a public value? Do we expect users to do something with it programmatically?

Reply:
I think I copied this (and doing the same for the thread pool size) from `NonBlockingFileIO`: https://github.com/apple/swift-nio/blob/6cac95501b52884d6ac9bf63f9029d983b179512/Sources/NIO/NonBlockingFileIO.swift#L32
but thinking about it more in their case they require the user to manage the thread pool so it makes more sense to have it programmatically available. I don't really know what people would do with this in our situation though so I'm fine with removing both of the constants